run.sh

```diff
@@ -1,67 +0,0 @@
-#!/bin/bash
-#
-# Miscellaneous scripts.
```

Comment:
What IS this file?! I have many times mistakenly opened it looking for how to run something, only to see it and then remember that is was totally useless. At it's core it looks like it's basically "short hand" for the shell one-liner:
cp -r _tmp/*report* ../rappor-gh-pages/examples && cp -r _tmp/doc ../rappor-gh-pages/doc
Which is clearly for exporting but to where? We don't check anything like that into github. What does it mean in the comments "commit and push"?

In any event this doesn't look like the sort of thing that someone using this would want or need. (I certainly haven't thus far) So I'd just assume delete it to get it out of the top level directory adding confusion.